spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -211,4 +211,10 @@ public boolean handleTimestampWithoutZone() {
         .defaultValue(SparkSQLProperties.HANDLE_TIMESTAMP_WITHOUT_TIMEZONE_DEFAULT)
         .parse();
   }
+
+  public Long streamFromTimestamp() {
+    return confParser.longConf()
+        .option(SparkReadOptions.STREAM_FROM_TIMESTAMP)
+        .parseOptional();
```

Comment:
May be better to put the default for fromTimetstamp here (currently -1 but should probably be Long.min_value) rather than passing a null through to InitialOffset and having InitialOffset setting the default

Reply:
I have put the default value as `Long.MIN_VALUE`.